src/renderer/features/settings/components/general/control-settings.tsx

```diff
@@ -192,6 +192,28 @@ export const ControlSettings = () => {
       isHidden: !isElectron(),
       title: 'Save play queue',
     },
+    {
+      control: (
+        <Switch
+          defaultChecked={settings.savePassword}
+          onChange={(e) => {
+            if (!e.currentTarget.checked) {
+              localSettings?.passwordClear();
+            }
+            setSettings({
+              general: {
+                ...settings,
+                savePassword: e.currentTarget.checked,
+              },
+            });
+          }}
+        />
+      ),
+      description:
+        'Save server passwords (encrypted by OS). Deselecting this will immediately remove any saved passwords',
+      isHidden: !isElectron(),
+      title: 'Save passwords',
+    },
```

Comment:
This feels a bit awkward in the actual settings page. The toggle should be moved to the "Manage servers" modal alongside the other 2 toggles.